src/router.ts

```diff
@@ -34,11 +34,15 @@ export class OneSchemaRouter<
 
   private constructor(
     private schema: Schema,
-    { introspection, using: router }: OneSchemaRouterConfig<R>,
+    public config: OneSchemaRouterConfig<R>,
```

Comment:
nit: since instance vars are public by default, we could omit `public` here, right?

Reply:
True, but I am making it private per Swain's comment above.